azure-pipelines.yml

```diff
@@ -61,7 +61,107 @@ variables:
 jobs:
   - template: tools/azure-pipelines/jobs-template.yml
     parameters: # see template file for a definition of the parameters.
-      stage_name: ci_build
```

Comment:
you need to put these changes into `tools/azure-pipelines/build-apache-repo.yml`.

Reply:
Thanks, I adjusted it as well